include/identity.php

```diff
@@ -663,6 +668,10 @@ function advanced_profile(App $a) {
 			$profile['age'] = array( t('Age:'), $age );
 		}
 
+		if ($age = age($a->profile['dob'], $a->profile['timezone'], '')) {
```

Comment:
Why are you rebasing against master when the development branch is the one that gets active changes? For me that sounds like the origin of all your reverts in the code.